docs/content/dagster-cloud/managing-deployments/managing-multiple-projects-and-teams.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Managing multiple projects & teams in Dagster Cloud
+description: "How to set up and manage multiple projects with Dagster Cloud Hybrid deployments."
+---
+
+# Managing multiple projects & teams in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Whether to manage complexity or to divide your work into areas of responsibility, you may want to isolate your code bases into multiple projects. See [Strategies for code separation](#strategies-for-code-separation) for more information.
+
+If you require isolation between your execution context, see [Strategies for execution context isolation](#strategies-for-execution-context-isolation).
+
+## Strategies for code separation
+
+<Note>
+  Disambiguation: In this section, repository refers to a depot under a version
```

Comment:
"depot" is not a familiar term to me in this context

consider s"refers to a version control system"

Reply:
FWIW i don't think i've seen a single user go by not on git - wonder if we should just assume they're on git for the purpose of this doc, call it "git repository" throughout the doc, and drop the disclaimer? (with apologies to mercurial/perforce users)

Reply:
trading one contextual definition for presumption with an apology? idk, @erinkcochran87 what do you think?